rewrite-maven/src/main/java/org/openrewrite/maven/AddPlugin.java

```diff
@@ -82,6 +83,12 @@ public class AddPlugin extends Recipe {
     @Nullable
     String filePattern;
 
+    @Option(displayName = "Add to the root pom",
+            description = "Add to the root pom where root is the eldest parent of the pom within the source set.",
+            required = false)
+    @Nullable
+    Boolean addToRootPom;
```

Comment:
I don't agree that this should be an option. In the presence of a multimodule project, always add it to the root pom. 

I don't think it would ever make sense to run this recipe with this being set to false as running the recipe in that configuration on a mixture of single and multi-module projects would always do the wrong thing on some set of repositories.

Reply:
I've pushed a commit to fix this